include/benchmark/benchmark.h

```diff
@@ -354,33 +405,16 @@ class BenchmarkReporter {
   // benchmark, thus have the same name.
   virtual void ReportRuns(const std::vector<Run>& report) const = 0;
 
-  virtual ~BenchmarkReporter() {}
+  virtual ~BenchmarkReporter();
```

Comment:
does this do something now? if not, an empty implementation in the header should be fine.